routers/init.go

```diff
@@ -190,6 +190,8 @@ func NormalRoutes() *web.Route {
 		r.Mount("/api/packages", packages_router.CommonRoutes())
 		// This implements the OCI API (Note this is not preceded by /api but is instead /v2)
 		r.Mount("/v2", packages_router.ContainerRoutes())
+		// Arch package routes
+		r.Mount("/api/packages/arch", packages_router.ArchRoutes())
```

Comment:
This should be in `CommonRoutes` and follow the structure of the other package types.